RevenueCatUI/CustomerCenter/Views/UIKit Compatibility/CustomerCenterViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CustomerCenterViewController.swift
+//
+//  Created by Will Taylor on 12/6/24.
+
+import RevenueCat
+import SwiftUI
+
+#if canImport(UIKit) && os(iOS)
+
+/// Warning: This is currently in beta and subject to change.
+///
+/// A SwiftUI view for displaying a customer support common tasks
```

Comment:
```suggestion
/// A UIKit ViewController for displaying a customer support common tasks
```

Reply:
also that docstring's copy could generally be improved, "displaying a customer support common tasks" sounds broken. 

I realize that this is likely copy-pasted from SwiftUI, maybe we do a follow-up PR improving both sides